services/scraper.js

```diff
@@ -12,11 +12,20 @@ const metascraper = require('metascraper');
 const scraper = {
 
   /**
-   * Create the new Task kue.
+   * Create the new Task kue singleton.
    */
-  task: new kue.Task({
-    name: 'scraper'
-  }),
+  _task: null,
```

Comment:
I think `_task` should be kept internal like:

let taskInstance = null;

const scraper = {
get task() {
    if (taskInstance) {
      return taskInstance;
    }

    taskInstance = new kue.Task({
      name: 'scraper'
    });

    return taskInstance;
  },
};